views/skala3ma-privacy.html

```diff
@@ -1,152 +1,42 @@
-<br>
-<br>Email:
-<br>
-<br>Address
-<br>
-<br>
-
-
-
-</div></div>
-
 
+					
+					<head>
+						<title>Protection des Données - FSGT Montagne-Escalade</title>
+					</head>
+					<body>
+						<h1>Protection de vos données personnelles : Engagement de la FSGT Montagne-Escalade</h1>
+						<p>
+							À la <strong>FSGT Montagne-Escalade</strong>, nous accordons une importance primordiale à la confidentialité et à la protection des données personnelles de nos membres, donateurs, et visiteurs. Conformément au Règlement Général sur la Protection des Données (RGPD) et à la législation en vigueur en matière d'informatique et de libertés, nous nous engageons sur les points suivants :
+						</p>
+						<ol>
+							<li><strong>Transparence</strong> : Nous collectons vos données de manière transparente, en vous informant clairement des finalités pour lesquelles elles sont traitées. Les données recueillies ici le sont uniquement dans le cadre des de l'organisation des <em>compétitions autrement</em> afin de calculer les classements des participants et participantes selon leur catégorie. </li>
+							<li><strong>Limitation de la collecte</strong> : Les données recueillies sont strictement nécessaires à la réalisation des <em>compétitions autrement</em>. Nous ne collectons pas de données superflues.</li>
+							<li><strong>Sécurité et Confidentialité</strong> : Nous assurons la sécurité de vos données personnelles par des mesures techniques et organisationnelles adaptées pour prévenir toute perte, utilisation non autorisée, divulgation ou modification.</li>
+							<li><strong>Consentement</strong> : Votre consentement est systématiquement sollicité pour tout traitement de données à caractère personnel, et vous pouvez le retirer à tout moment.</li>
+							<li><strong>Droits d'accès, de rectification et d'effacement</strong> : Vous avez le droit de consulter, modifier, ou demander la suppression de vos données personnelles. Pour cela, vous pouvez nous contacter à <a href="mailto:competitions-autrement@listes.montagne.fsgt.org">competitions-autrement@listes.montagne.fsgt.org</a>.</li>
+							<li><strong>Durée de conservation</strong> : Les données sont conservées pendant une durée n'excédant pas celle nécessaire aux fins pour lesquelles elles ont été collectées ou traitées.</li>
+							<li><strong>Partage avec des tiers</strong> : Nous nous engageons à ne pas partager vos données avec des tiers sans votre consentement explicite, sauf obligation légale.</li>
+						</ol>
+						<p>
+							Pour toute question concernant la protection de vos données personnelles, ou pour exercer vos droits, n'hésitez pas à nous contacter à <a href="mailto:competitions-autrement@listes.montagne.fsgt.org">competitions-autrement@listes.montagne.fsgt.org</a>.
+						</p>
+					</body>
+					
+					
 
```

Comment:
I have to check with google play store rules if they will accept privacy notice in French.. we might need to  just fix the english version instead of removing it. on top of the notice we can use in page anchors to jump to the english version

Reply:
I suggest to translate the current French page to English and have a way to display the correct page depending on the language (not sure yet how to do...)

Reply:
Ok I did it and pushed the changes. I also fixed the footer layout.